### ISSUE TITLE
Platform: Unified Remote and Mousepad

### ISSUE THREAD

**Issue by Nerwyn** [Nerwyn/android-tv-card#124]
- Add new platform Unified Remote using [this custom integration](https://github.com/DaviPtrs/hass-unified-remote).
- Add a new mousepad element for use with this integration.

**Comment by Nerwyn** [comment]
Keyboard support and basic default keys have been added in the 4.3.0 alpha.

For mouse support, I found [this bit of code](https://github.com/unifiedremote/Remotes/blob/master/Main/Basic%20Input/remote.lua#L41-L44) which can be called using the following command:

```yaml
action: unified_remote.call
data:
  target: desktop
  remote_id: Relmtech.Basic Input
  action: delta
  extras:
    Values:
      - Value: 0
      - Value: 200
      - Value: 200
```

**Comment by Nerwyn** [comment]
[Core.Input](https://github.com/unifiedremote/Remotes/blob/329d04f3c32b29038b0a6bc1ba034eefaff04267/Core/Input/remote.lua#L21) is a better API to use.

**Comment by Nerwyn** [comment]
Mousepad and ADB:

[This article](http://blog.blecentral.com/2015/07/01/simulate-user-generated-events-in-android/) describes adb input events I can use.

```
adb shell input trackball press
adb shell input trackball roll {{ diffX }} {{ diffY }}
```

**Comment by Nerwyn** [comment]
[Released 4.3.0](https://github.com/Nerwyn/android-tv-card/releases/tag/4.3.0)
